### DISCUSSION TITLE
My C #. net8 webAPI project cannot write logs to seq using log4net. Can you help me check?

### DISCUSSION THREAD

**Discussion by deepinnova** [datalust/seq-tickets#2285] [Help]
This is my project, how do I configure log4net
![image](https://github.com/user-attachments/assets/edd0d85c-49fd-4a58-a18a-074101d3641c)

![image](https://github.com/user-attachments/assets/2522b16b-06d5-46d5-a2e0-4dc9c80c3343)

![image](https://github.com/user-attachments/assets/0798576c-15aa-4bdb-ba3a-bbf625132f2a)

I have been using log4net in my project for a long time and it is very user-friendly. My company has used log4net in multiple projects, and today I wanted to use the powerful logging function of seq. However, I found that the logs cannot be written and there are no task prompts. The logs can be written to the console;
I use C #. net8 serilog seq, which can completely write logs to seq, but my company's project is too complicated and I don't want to convert log4net to serilog. Please help me, thank you!

**Comment by nblumhardt** [comment]
Hi! Turning on Log4Net's and collecting internal debugging would be the first thing to try:

https://logging.apache.org/log4net/release/faq.html#internalDebug

I'd also suggest turning off/skipping the async appender configuration until you have the plain direct appender working.

If that's still no help, checking Seq's ingestion log (Settings > Diagnostics) is the next place to look, followed by using an HTTP debugging proxy like Fiddler to check whether the application is trying to send data to Seq and where that might be failing.

Hope this helps,
Nick
